trezor-crypto/crypto/tests/test_check.c

```diff
@@ -4990,8 +4990,9 @@ START_TEST(test_mnemonic) {
   a = vectors;
   b = vectors + 1;
   c = vectors + 2;
+  char buf[240];
   while (*a && *b && *c) {
-    m = mnemonic_from_data(fromhex(*a), strlen(*a) / 2);
+    m = mnemonic_from_data(fromhex(*a), strlen(*a) / 2, buf, 240);
```

Comment:
should we use `MnemonicBufLength` here instead?

Reply:
As a general idea, tests should not use consts from product code, that can hide errors, better to use it own constant values.